plugins/carbon_black_defense/help.md

```diff
@@ -319,8 +373,10 @@ When selecting adjusting frequency please review [rate limits](https://developer
 Get Notifications trigger requires that the [API key type](https://developer.carbonblack.com/reference/carbon-black-cloud/authentication/) is a 'SIEM' type.
 The Role Based Access Control (RBAC) of your API Key will require the right permissions as noted in the following link: [https://developer.carbonblack.com/reference/carbon-black-cloud/cb-defense/latest/platform-search-api-enriched-events/#authentication](https://developer.carbonblack.com/reference/carbon-black-cloud/cb-defense/latest/platform-search-api-enriched-events/#authentication).
 
+The Find Event API and Get Details for Specific Event API are now documented as Enriched Events Search and Enriched Event Details Search at the following links: [https://developer.carbonblack.com/reference/carbon-black-cloud/cb-defense/latest/platform-search-api-enriched-events/#start-an-enriched-events-search-v2](https://developer.carbonblack.com/reference/carbon-black-cloud/cb-defense/latest/platform-search-api-enriched-events/#start-an-enriched-events-search-v2) and [https://developer.carbonblack.com/reference/carbon-black-cloud/cb-defense/latest/platform-search-api-enriched-events/#request-details-for-enriched-events-v2](https://developer.carbonblack.com/reference/carbon-black-cloud/cb-defense/latest/platform-search-api-enriched-events/#request-details-for-enriched-events-v2).
 # Version History
 
+* 3.0.0 - Update Get Event Details Action and Find Event Action
```

Comment:
```suggestion
* 3.0.0 - Update Get Event Details Action and Find Event Action supporting new API. Update SDK to version4
```

Reply:
Updated to include API and SDK update mentions.